script.js

```diff
@@ -0,0 +1,32 @@
+const rock = "rock";
+const paper = "paper";
+const scissors = "scissors";
+const compVal = ['rock','paper','scissors'];
+let humanVal;
+
+function computerPlay() {   
+    let rand = Math.floor(Math.random()*compVal.length);
```

Comment:
That look like a `const` to me, also please use a full name for a variable, so `randomValue` in this case, instead of `rand`

Reply:
what looks like a constant? variable name corrected